src/plugins/analysis/crypto_hints/signatures/additional_signatures.yara

```diff
@@ -0,0 +1,111 @@
+        // precalculated AES decryption lookup table
+        $dec_lt = { 51 f4 a7 50 7e 41 65 53 1a 17 a4 c3 3a 27 5e 96 3b ab 6b cb 1f 9d 45 f1 ac fa 58 ab 4b e3 03 93 }
+    condition:
+        any of them
+}
+
+rule SMIME_IDs {
+	meta:
+		description = "Cipher S/MIME object identifiers (RFCs 3447 & 5754)"
+    strings:
+        $md2 = { 2a 86 48 86 f7 0d 02 02 05 }
+        $md5 = { 2a 86 48 86 f7 0d 02 05 05 }
+        $sha1 = { 2b 0e 03 02 1a 05 00 04 14 }
+        $sha256 = { 60 86 48 01 65 03 04 02 01 }
+        $sha384 = { 60 86 48 01 65 03 04 02 02 }
+        $sha512 = { 60 86 48 01 65 03 04 02 03 }
+        $sha224 = { 60 86 48 01 65 03 04 02 04 }
+
+        $dsa_sha224 = { 60 86 48 01 65 03 04 03 01 }
+        $dsa_sha256 = { 60 86 48 01 65 03 04 03 02 }
+
+        $rsa_sha224 = { 06 09 2a 86 48 86 f7 0d 01 01 0e 05 00 }
+        $rsa_sha256 = { 06 09 2a 86 48 86 f7 0d 01 01 0b 05 00 }
+        $rsa_sha384 = { 06 09 2a 86 48 86 f7 0d 01 01 0c 05 00 }
+        $rsa_sha512 = { 06 09 2a 86 48 86 f7 0d 01 01 0d 05 00 }
+
+        $ecdsa_sha224 = { 06 08 2a 86 48 ce 3d 04 03 01 }
+        $ecdsa_sha256 = { 06 08 2a 86 48 ce 3d 04 03 02 }
+        $ecdsa_sha384 = { 06 08 2a 86 48 ce 3d 04 03 03 }
+        $ecdsa_sha512 = { 06 08 2a 86 48 ce 3d 04 03 04 }
+    condition:
+        any of them
+}
+
+rule Tiger_Hash_Constants {
+	meta:
+		description = "Tiger hash substitution box constants"
+    strings:
+        $c1 = { 5E 0C E9 F7 7C B1 AA 02 }
+        $c2 = { EC A8 43 E2 03 4B 42 AC }
+        $c3 = { D3 FC D5 0D E3 5B CD 72 }
+        $c4 = { 3A 7F F9 F6 93 9B 01 6D }
+        $c5 = { 93 91 1F D2 FF 78 99 CD }
+        $c6 = { E2 29 80 70 C9 A1 73 75 }
+        $c7 = { C3 83 2A 92 6B 32 64 B1 }
+        $c8 = { 70 58 91 04 EE 3E 88 46 }
+        $c9 = { 38 21 A1 05 5A BE A6 E6 }
+        $c10 = { 98 7C F8 B4 A5 22 A1 B5 }
+        $c11 = { 90 69 0B 14 89 60 3C 56 }
+        $c12 = { D5 5D 1F 39 2E CB 46 4C }
+        $c13 = { 34 94 B7 C9 DB AD 32 D9 }
+        $c14 = { F5 AF 15 20 E4 70 EA 08 }
+        $c15 = { F1 8C 47 3E 67 A6 65 D7 }
+        $c16 = { 99 8D 27 AB 7E 75 FB C4 }
+    condition:
+        4 of them
+}
+
+rule camellia_constants {
+	meta:
+		description = "Camellia cipher substitution table constants"
+    strings:
+        $c1 = { 70 82 2C EC B3 27 C0 E5 E4 85 57 35 EA 0C AE 41 }
+        $c2 = { E0 05 58 D9 67 4E 81 CB C9 0B AE 6A D5 18 5D 82 }
+        $c3 = { 38 41 16 76 D9 93 60 F2 72 C2 AB 9A 75 06 57 A0 }
+        $c4 = { 70 2C B3 C0 E4 57 EA AE 23 6B 45 A5 ED 4F 1D 92 }
+    condition:
+        all of them
+}
+
+rule present_cipher {
+	meta:
+		description = "PRESENT block cipher substitution table constants"
+    strings:
+        // substitution box
+        $sb = { 0C 05 06 0B 09 00 0A 0D 03 0E 0F 08 04 07 01 02 }
+        // inverse substitution box
+        $isb = { 05 0E 0F 08 0C 01 02 0D 0B 04 06 03 00 07 09 0A }
+    condition:
+        all of them
+}
```

Comment:
There is mixed indentation-depth and -types in this file. Could we reformat please? :-)